docs/getting-started.md

```diff
@@ -226,57 +226,104 @@ With an extension created, install the toolkit package using this command:
 npm install --save @vscode/webview-ui-toolkit
 ```
 
-### Using the toolkit inside a webview
+### Register the toolkit components
 
-With the package installed, you need to adjust the project so the toolkit is usable within your webview. To do this, start by updating the `_getWebviewContent` method to accept two new parameters.
+It can be helpful to think of webviews as having a frontend and a backend.
 
-```typescript
-// file: src/panels/HelloWorldPanel.ts
+In the first part of this guide, you created the backend of a webview (i.e. `HelloWorldPanel`) and in this part, you'll create and configure the frontend JavaScript of a webview.
```

Comment:
I wonder how, if at all, we can better explain why we are framing this as a backend concept given that `HelloWorldPanel` sounds like UI.

Reply:
Agreed. This is something I'm 50-50 on including and wanted to come back to.